addons/website_slides/models/slide_channel.py

```diff
@@ -519,12 +531,13 @@ def message_post(self, *, parent_id=False, subtype_id=False, **kwargs):
 
     def action_redirect_to_members(self, state=None):
         action = self.env["ir.actions.actions"]._for_xml_id("website_slides.slide_channel_partner_action")
-        action['domain'] = [('channel_id', 'in', self.ids)]
+        action_ctx = {'search_default_channel_id': self.ids}
```

Comment:
Let's pass this action into a self.ensure_one as well because I don't know if search_default_channel_id with multiple ids is correctly supported.

(So we will be able to remove the "len(self) == 1" thing too)